chaos_genius/views/anomaly_data_view.py

```diff
@@ -431,8 +436,13 @@ def _get_dimensions_values(
         {
             "label": dimension,
             "value": dimension,
+            "label_path_safe": make_path_safe(dimension),
             "subdim_value_options": [
-                {"label": value, "value": value}
+                {
+                    "label": value,
+                    "value": value,
+                    "label_path_safe": make_path_safe(value),
```

Comment:
Is this being used at any other place?

Reply:
It is used by the frontend to make the CSV name and nowhere else.